tests/test_executor/test_executors.py

```diff
@@ -13,63 +13,93 @@
 
 
 class TestExecutor(TestEGCG):
+    executor_cls = Executor
+
+    def execute(self, *args, **kwargs):
+        e = self.executor_cls(*args, **kwargs)
+        try:
+            e.start()
+        except NotImplementedError:
+            pass
```

Comment:
@mwhamgenomics It might be a good idea to log any passes like this in a debug or error log, since it would be a good idea to keep an eye on such instances.

Reply:
I doubt if that would be useful - the only reason we'd hit an exception here is if e is a direct instance of `Executor`, which indeed doesn't implement `start`.

Probably is a bit messy to use exceptions here, though - can use `isinstance`, etc. instead. Or can avoid raising NotImplementedError in `Executor`, since it's not an abstract class anyway